### PR TITLE
Identify conflicts on carry commits

### PR DESCRIPTION
**Pull request by kargakis**
Depends on adopting the rebase flow in OSE.

@bbguimaraes @jupierce @stevekuznetsov

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -37,15 +39,41 @@ Jenkins job: ${env.BUILD_URL}
...
+    try {
+        sh("""\
+git config user.name jenkins
+git config user.email jenkins@example.com
+git checkout ${branch}
+git rebase '${commit}'
+""")
+    } catch (err) {
+        sh("""\
+CONFLICTING_COMMIT=$(cat .git/rebase-apply/original-commit)
```
Comment (stevekuznetsov) [inline Jenkinsfile]: This seems hack-ey -- is there a way to look this up using the plumbing commands?

Comment (kargakis) [inline Jenkinsfile]: The plumbing commands?

Comment (stevekuznetsov) [inline Jenkinsfile]: https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain

Comment (kargakis) [inline Jenkinsfile]: Interesting read. Surprisingly, no plumbing command for rebasing ...

**Pull request closed without merging**